Bottom: 80abe6bbe3656ee161221635e59203f8016e9edc
Top:    8a9be61bab48ea35140dedd4ce20dbb78504beef
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 18:51:18 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
